templates/cloudflare-workers/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "scripts": {
     "dev": "wrangler dev src/index.ts",
```

Comment:
Can we remove `src/index.ts` here?

```js
wrangler dev
```

Reply:
Indeed it is. Sorry.

Reply:
@yusukebe 
Fixed
[a2afa6c](https://github.com/honojs/starter/pull/52/commits/a2afa6c3e536914ff5b42eed2f9b618a91efc817)